modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -600,24 +600,25 @@ private CompletableFuture<?> onTableCreate(ConfigurationNotificationEvent<TableV
             CatalogTableDescriptor tableDescriptor = toTableDescriptor(ctx.newValue());
             CatalogZoneDescriptor zoneDescriptor = getZoneDescriptor(tableDescriptor.zoneId());
 
-            List<Set<Assignment>> assignments;
+            CompletableFuture<List<Set<Assignment>>> assignments;
```

Comment:
It would be better to rename this variable to `assignmentsFuture`, as well as the parameter in methods.

Reply:
Renamed.